usmqe_tests/rpm/test_install.py

```diff
@@ -8,12 +8,13 @@
 
 
 @pytest.mark.parametrize("rpm_name", [
-    "tendrl-commons",
-    "tendrl-node-agent",
-    "tendrl-ceph-integration",
-    "tendrl-gluster-integration",
     "tendrl-api",
```

Comment:
Could you make sure that `tendrl-commons` is a first item there? It would be better wrt sharing rpm chroot tree where the packages are installed.